corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_property.html

```diff
@@ -159,6 +159,14 @@ <h4 class="modal-title">
                           <input type="checkbox" data-bind="checked: hidden" />
                         </div>
                       </div>
+                      <div class="form-group">
```

Comment:
Am I reading right that this is inside of the `has_geocoder_privs` block? That doesn't seem right.

Since this feature is going to require a mobile change, it should be limited based on CommCare version. The usual way to do this is to add a function to [feature_support.py](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/feature_support.py) that's named after the new feature and limits to the current CC dev version (2.53). That code can also check the `USH_CASE_CLAIM_UPDATES` flag.

Reply:
You're right, that seems wrong. Adding `enable_exclude_from_search` to `feature_support.py`.